### DISCUSSION TITLE
Allow deleting broken dependencies

### DISCUSSION THREAD

**Discussion by Manenderrr** [godotengine/godot-proposals#10798] [Editor]
If you delete multiple files that were used in other places, it causes very large amount of scenes to be broken. Since all that data was removed from the project and you are probably going to remake it from scratch, you don't really have anything to fix all the broken dependecies with, meaning you will either have to look for every single file that uses one of the files you want to remove, or replace all missing dependencies with placeholder files and then replace all of the placeholders with something meaningful. Both workarounds are very cumbersome.

What I suggest is to allow completely removing the dependencies while on the dependency screen, removing all missing scene instances and resources and allowing opening the scene without having to satisfy every dependency.

(I don't have a picture for this, I guess a trash bin button next to the replacement button would be good enough)

**Comment by Aruudlay** [comment]
This is the bane of my life, would love the ability to delete dependencies
